docs/index.rst

```diff
@@ -1,8 +1,16 @@
 ********************
-ndcube Documentation
+ndcube Guide
 ********************
 
-This is the documentation for ndcube.
+Welcome to the ndcube User Guide. ndcube is a SunPy-affiliated
+package designed for handling n-dimensional datacubes described
+by a WCS (World Coordinate System) translation.
+
+.. toctree::
+   :maxdepth:3
+	     introduction
```

Comment:
indentation should match the maxdepth line

Reply:
As in ```introduction``` should be moved left to under the first colon before maxdepth?  Or vice versa?  Or does it matter?

Reply:
introduction should be shifted to the left.